docs/vNext/FISCO_BCOS_Configuration.md

```diff
@@ -0,0 +1,171 @@
+
+### Adaptor Code
+
+The FISCO BCOS adaptor code can be found in the following folders:
+
+- `caliper/packages/caliper-samples/benchmark/fisco-bcos/`, contains three benchmark scripts and their configuration files::
+  - `helloworld`, a basic benchmark for testing getting/setting a string.
+  - `transfer/solidity`, a benchmark for testing transferring money between different accounts. Its smart contract is implemented in [solidity](https://solidity.readthedocs.io/en/latest/) programming language
+  - `transfer/precompiled`, a benchmark for testing transferring money between different accounts. Its smart contract is implemented by the [precompiled contract framework](https://fisco-bcos-documentation.readthedocs.io/en/latest/docs/design/virtual_machine/precompiled.html)  of FISCO BCOS.
+- `caliper/packages/caliper-samples/network/fisco-bcos/`, contains example configuration files for network configuration.
+- `caliper/packages/caliper-samples/src/contract/fisco-bcos/`, contains smart contracts used by benchmarks.
+- `caliper/packages/caliper-fisco-bcos/`, the code of FISCO BCOS adapter implementing the Caliper interfaces.
+
+### Node Configuration
+
+A FISCO BCOS node uses several configuration files to control the network settings and certificates for connecting verification, which are described below:
+
+<font size=2>*\*we refer the root directory of node as `node/`*</font>
+| Name | Location | Function |
+| :--- | :---- | :---- |
+| config.ini | `node/` | `ini` format, contains network ports (PRC, P2P and channel), black list of nodes, path of data and certificates, etc.
+| group.***n***.genesis | `node/conf/` | `ini` format, the ***fixed configuration*** of group ***n***'s genesis block. FISCO BCOS support running several uncorrelated groups on the same set of nodes, you can get more information about concept "group" from [introduction of group architecture](https://fisco-bcos-documentation.readthedocs.io/en/latest/docs/design/architecture/group.html). The configuration contains the type of consensus algorithm,  ID of initial nodes, etc.|
+| group.***n***.ini | `node/conf/` | `ini` format, the ***changeable configuration*** of group ***n***,  includes parameters of consensus module and storage module, configuration for features, etc. |
+| node.key | `node/conf/` | `PEM` format, the private key used to connect to other nodes via SSL/TLS, using ECDSA-secp256k1. |
+| node.crt | `node/conf/` | the certificate used in SSL/TLS connection between nodes. |
+| ca.crt | `node/conf/` | the root certificate used in SSL/TLS connection between nodes. The ca.crt is used to verify whether the cerficate of peer node is correct or not. |
+
+For a complete example, please refer to the example files in  `caliper/packages/caliper-samples/network/fisco-bcos/4nodes1group/node*`.
```

Comment:
Please remove as this is a maintenance issue ... we will be removing the folder very soon!